devito/operator/operator.py

```diff
@@ -188,7 +189,8 @@ def _sanitize_exprs(cls, expressions, **kwargs):
 
         for i in expressions:
             if not isinstance(i, Evaluable):
-                raise InvalidOperator("`%s` is not an `Evaluable` object" % str(i))
+                raise CompilationError(f"`{i!s}` is not an Evaluable object, "
+                                       "check again your Equation")
```

Comment:
Nitpick: "check your equation again" would be more natural phrasing, and the comma should probably be a semicolon

Reply:
fixing now